packages/net/writebuffer.pony

```diff
@@ -0,0 +1,69 @@
+use "collections"
+
+class WriteBuffer
+  """
+  A simple constant sized byte buffer to push Numbers into.
+  The data container is iso in order to keep it sendable(?)
+  Potential upgrades:
+  1. Have a list of data buffers of preset size
+  2. Arguably, seperate functions for _be and _le
+  3. A better way to get data out of the buffer, without passing a new array?
```

Comment:
Can you include example code in the doc string for how you would use this class? See signals and bureaucracy packages for examples.